webapp/src/main/webapp/iaf/gui/js/config.js

```diff
@@ -101,72 +101,41 @@ angular.module('iaf.beheerconsole').config(['$cookiesProvider', '$locationProvid
 			},
 		},
 	})
-	.state('pages.errorstorage', {
+	.state('pages.storage', {
 		abstract: true,
 		url: "/adapter/:adapter/:receiver/",
```

Comment:
Could you rename this to `/adapter/:adapter/receiver/:receiver/` the the url is somewhat api compliant?